nrf_rpc: Doc review from oyvindronningstad (#1)
doki-nordic/nrfxlib: nrf_rpc/doc/architecture.rst

Events always reserve a new thread from the remote thread pool.
Pay special attention when sending multiple events one after another, because each event reserves a new thread.

> Sending events too fast might consume the entire thread pool and, as a result, block all outgoing commands and events until any in the thread pool thread becomes available.

Author: Dodałem `until`, żeby mieć pewność, że nikt tego nie zrozumie w ten sposób, że są one zablokowane bez końca.

> Sending events too fast might consume the entire thread pool and, as a result, block all outgoing commands and events until any in the thread pool thread becomes available.

Reviewer: ```suggestion
Sending events too fast might consume the entire thread pool and, as a result, block all outgoing commands and events until a thread becomes available in the thread pool .
```

Two kinds of errors might occur when using this library.

 Error during parsing of an incoming packet

>     These kinds of error cannot be directly returned to the user as a return value.

Author: Usunąłbym `e.g.`, bo tu tylko wchodzi w grę `return value`, więc to nie jest przykład, bo innej możliwości nie przewiduję.

>     These kinds of error cannot be directly returned to the user as a return value.

Reviewer: ```suggestion
    These errors cannot be directly returned to the user as a return value.
```

    The user is informed about such errors through a callback.
    First, if the group in which the error happened is known, then a group error callback is called.
    Second, a global error handler (provided to the :cpp:func:`nrf_rpc_init` function) is called.

>     A malformed packets should not normally happen, because the transport layer is responsible for reliable packet transferring.

Reviewer: ```suggestion
    Malformed packets should not normally happen, because the transport layer is responsible for reliable packet transferring.
```

Operating system abstraction
----------------------------

> The operating system abstraction provides functionality for nRF RPC that depends on an operating system.

Reviewer: ```suggestion
The operating system abstraction provides functionality for nRF RPC that depends on the operating system.
```
